_docs/development.md

```diff
@@ -25,23 +25,26 @@ You'll need the following to get started:
 ## Create temba user for PostgreSQL
 
 {% highlight bash %}
-$ createuser temba --pwprompt -d
+$ sudo apt-get install postgresql-client postgresql-server-dev-9.3 postgis*
+$ sudo -u postgres createuser temba --pwprompt -d
 Enter password for new role:
 Enter it again:
+$ sudo adduser temba
```

Comment:
You're creating a system user for temba here to make authentication easier for postgres, but I don't think that is actually required.